src/main/kotlin/com/carneirofatima/vehicleterminal/dao/DriverDao.kt

```diff
@@ -0,0 +1,13 @@
+package com.carneirofatima.vehicleterminal.dao
+
+import com.carneirofatima.vehicleterminal.entity.Driver
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.data.jpa.repository.Query
+import org.springframework.stereotype.Repository
+
+@Repository
+interface DriverDao : JpaRepository<Driver, Int> {
+
+    @Query(value = "SELECT register FROM Driver d WHERE d.register = :register")
```

Comment:
Why you need `@Query`? Spring Data can resolve for us only with method name
Read this: https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#jpa.query-methods

Reply:
```
fun existsByRegister(register: String): Boolean
```